spectrum_utils/plot.py

```diff
@@ -5,11 +5,11 @@
 
 import matplotlib.pyplot as plt
 import matplotlib.ticker as mticker
+import numpy as np
 
 import spectrum_utils.fragment_annotation as fa
 from spectrum_utils.spectrum import MsmsSpectrum
 
-
```

Comment:
**todo:** Add the second blank line again according to [PEP8](https://peps.python.org/pep-0008/).

Reply:
A case of Black vs PEP8 I never knew 😅 I'll add it again!

Reply:
Does black complain about this on your end? For me it leaves it unchanged.

PEP8 is maybe a bit ambiguous on the number of blank lines before statements, but I just use 2 blank lines to be consistent with subsequent method or class definitions.

Reply:
Black at least changes it like this for me. I added the second line again. We'll see what the Black linter says in CI.